js/app-super-resolution/README.md

```diff
@@ -0,0 +1,59 @@
+# RUNNING SUPER-RESOLUTION ON IMAGES ACROSS VARIOUS PLATFORMS USING ONNX RUNTIME MODELS
```

Comment:
general suggestion for read.me: 

maybe can make it more complete follow the same pattern/sections from maui example:

https://github.com/microsoft/onnxruntime-inference-examples/blob/main/mobile/examples/Maui/MauiVisionSample/readme.md

maybe for sections like"pre/post processing" methods, can expand to explain in more details in github tutorial page, and briefly talk about it here would be good.

Reply:
adding that now